src/fast_check/mod.rs

```diff
@@ -533,7 +533,7 @@ pub fn build_fast_check_type_graph<'a>(
           let source_hash = graph
             .get(specifier)
             .and_then(|m| m.source())
-            .map(|s| fast_insecure_hash(s.as_bytes()))
+            .map(|s| fast_insecure_hash(s.get().unwrap().as_bytes()))
```

Comment:
I think it's preferable to unwrap to cause a panic in the analysis code because this not existing in the graph would be a bug.